torch/jit/_trace.py

```diff
@@ -1036,7 +1036,8 @@ def check_unique(param):
                 tmp_module._parameters[name] = param
                 check_unique(param)
         for name, buf in orig._buffers.items():
-            if buf is not None:
+            # If the buffer is marked as non persistent, then it is ignored
+            if buf is not None and name not in orig._non_persistent_buffers_set:
```

Comment:
I don't understand this line. Shouldn't non-persistent buffers still be added to traced modules? They are just like any other module attribute that does not get saved to the state dict, right?